Sources/DatadogSDKTesting/Objc/LibraryInitializer.m

```diff
@@ -0,0 +1,18 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2020-2021 Datadog, Inc.
+ */
+
+#import <Foundation/Foundation.h>
+#import <objc/runtime.h>
+
+__attribute__((constructor)) static void initialize_FrameworkLoadHandler() {
```

Comment:
we can leave a comment with [Apple docs link](https://developer.apple.com/documentation/objectivec/nsobject/1418815-load) as this line may require explanation

Reply:
Yes, good idea. Added comment